Src/FluentAssertions/Equivalency/EquivalencyValidator.cs

```diff
@@ -62,6 +66,19 @@ private static void TrackWhatIsNeededToProvideContextToFailures(AssertionScope s
         scope.TrackComparands(comparands.Subject, comparands.Expectation);
     }
 
+    private static void AssertComparandsPointToActualObjects(Comparands comparands)
+    {
+        if (ReferenceEquals(comparands.Subject, comparands.Expectation))
+        {
+            return;
+        }
+
+        if (comparands.Subject is null || comparands.Expectation is null)
```

Comment:
The ` || comparands.Expectation is null` part is not covered by a test.
Can we write one or is this path uncoverable?